### PR TITLE
chore: add webpack for embed browser build

### PR DESCRIPTION
**Pull request by mrisoli**
## Description

Add webpack, webpack-cli and ts-loadr as devDependencies for embed
Add webpack config for browser bundle
Add script to package.json to build with webpack

## Checklist

- [ ] Explain the **motivation** for making this change.
- [ ] Provide a solid **test plan** for your changes.
- [ ] Link to the issue - if any [issue number here]()

### CODE REVIEW

File: packages/embed/package.json [hunk 1]
```diff
@@ -3,13 +3,15 @@
   "version": "1.0.0",
   "private": true,
   "main": "build/index.js",
+  "browser": "dist/webpack/bundle.js",
```
Comment (prilutskiy) [inline packages/embed/package.json:6]: I'm curious, how does this work?

**Review by prilutskiy — APPROVED** [review]

File: packages/embed/package.json (hunk 1 above)
Comment (mathio) [reply packages/embed/package.json:6]: https://docs.npmjs.com/cli/v6/configuring-npm/package-json#browser

Comment (mrisoli) [reply packages/embed/package.json:6]: This is just a hint field, it doesn't directly do anything but hint that file is supposed to be used in the browser.

Comment (prilutskiy) [reply packages/embed/package.json:6]: Oh, okay, I thought it maybe automatically does something :)

**Review by mathio — APPROVED** [review]

**Pull request merged**